### PR TITLE
Rollup brief documentation

### PR DESCRIPTION
**Pull request by munrocket**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,33 @@ $ echo $?
 0
 ```
 
-## webpack
+## With rollup
```
Comment (juliangruber) [inline README.md]: to be consistent

```suggestion
## rollup
```

File: README.md [hunk 2]
```diff
@@ -42,7 +42,33 @@ $ echo $?
...
+
+In simple cases you can run it from command line
```
Comment (juliangruber) [inline README.md]:
```suggestion
In simple cases you can run `rollup` and `tape-run` right from command line:
```

File: README.md [hunk 3]
```diff
@@ -42,7 +42,33 @@ $ echo $?
...
+```bash
+$ rollup test/test.js -f iife  | tape-run
```
Comment (juliangruber) [inline README.md]: can you describe what `-f iife` does?

File: README.md [hunk 4]
```diff
@@ -42,7 +42,33 @@ $ echo $?
...
+```
+But maybe you need configuration file that write to stdout and started it with `rollup -c | tape-run` for example like that
```
Comment (juliangruber) [inline README.md]:
```suggestion
If you want to use a configuration file, here's an example for `rollup -c | tape-run`:
```

File: README.md [hunk 5]
```diff
@@ -42,7 +42,33 @@ $ echo $?
...
+  output: { format: 'iife', sourcemap: 'inline' },
+  plugins: [
+    resolve(),
+    commonjs(),
+    builtins(),
+    istanbul({ exclude: ['dist'] })
+  ]
+}
+```
+Add plugins that you need too! Also you can save intermediate result from rollup somewhere if you set output file.
```
Comment (juliangruber) [inline README.md]:
```suggestion
```

This is not relevant, people can figure this out :)

File: README.md [hunk 6]
```diff
@@ -42,7 +42,33 @@ $ echo $?
...
 
-## webpack
+## With rollup
+
+In simple cases you can run it from command line
+```bash
+$ rollup test/test.js -f iife  | tape-run
+```
+But maybe you need configuration file that write to stdout and started it with `rollup -c | tape-run` for example like that
+```
```
Comment (juliangruber) [inline README.md]:
```suggestion
```js
```

**Comment by munrocket** [comment]
Sorry, switched to zora, merge as you want :D

**Comment by juliangruber** [comment]
Could you give me access to push to your branch?

File: README.md (hunk 3 above)
Comment (munrocket) [reply README.md]: it produce code for browser in `immediately invoke function expression`

**Comment by munrocket** [comment]
done

**Comment by juliangruber** [comment]
awesome thank you!

**Comment by juliangruber** [comment]
published as 6.0.1

**Pull request merged**